src/sentry/static/sentry/app/views/performance/transactionSummary/index.tsx

```diff
@@ -196,13 +182,10 @@ class TransactionSummary extends React.Component<Props, State> {
                 orgSlug={organization.slug}
                 location={location}
               >
-                {({tableData, isLoading}) => {
-                  if (isLoading) {
```

Comment:
@alxu-sentry Is this where the block was occurring?

Reply:
yes. if we don't render any subcomponents then they won't issue their `<DiscoverQuery>`s